psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -1611,6 +1611,13 @@ void OEProp::compute_mulliken_charges()
     outfile->Printf( "\n   Total alpha = %8.5f, Total beta = %8.5f, Total charge = %8.5f\n", \
         suma, sumb, nuc - suma - sumb);
     wfn_->set_atomic_point_charges(apcs);
+
+    auto vec_apcs = std::make_shared<Matrix>("Mulliken Charges: (a.u.)", 1, mol->natom());
+    for (size_t i = 0; i < mol->natom(); i++){
+        vec_apcs->set(0, i, (*apcs)[i]);
+    }
+    wfn_->set_array("MULLIKEN_CHARGES", vec_apcs);
+
```

Comment:
Since we are starting to use wfn arrays more heavily. It may be a good idea to provide overloads for `set_array` that accept `std::vector<double>`, `psi::Vector` and `*double` + `size`, and do the conversion to a 1 row matrix automatically. Just to that is does not need to be done over and over again.

Reply:
I would argue that we really need a N-dimensional tensor class before allowing folks to arbitrary overload `set_arrays` like that. Happy to take PR's on this however.